sio-digital-team/active-safe
# Tableau embed added with mobile responsive

https://trello.com/c/gQTCa8zp


**Changes:**

- [x] Replaced table image with tableau embed code
- [x] Added mobile responsiveness to hide table on mobile
- [x] Added some css to hide the toolbar from the embed
  

**How to test:**

- Go to the injury statistics section and click on the next slide to see the graph and the graph is loaded and interactive
